src/Microsoft.ML.StandardLearners/StandardLearnersCatalog.cs

```diff
@@ -173,12 +172,26 @@ public static class StandardLearnersCatalog
             float learningRate = AveragedLinearArguments.AveragedDefaultArgs.LearningRate,
             bool decreaseLearningRate = AveragedLinearArguments.AveragedDefaultArgs.DecreaseLearningRate,
             float l2RegularizerWeight = AveragedLinearArguments.AveragedDefaultArgs.L2RegularizerWeight,
-            int numIterations = AveragedLinearArguments.AveragedDefaultArgs.NumIterations,
-            Action<AveragedPerceptronTrainer.Arguments> advancedSettings = null)
+            int numIterations = AveragedLinearArguments.AveragedDefaultArgs.NumIterations)
         {
             Contracts.CheckValue(ctx, nameof(ctx));
             var env = CatalogUtils.GetEnvironment(ctx);
-            return new AveragedPerceptronTrainer(env, labelColumn, featureColumn, weights, lossFunction ?? new LogLoss(), learningRate, decreaseLearningRate, l2RegularizerWeight, numIterations, advancedSettings);
+            return new AveragedPerceptronTrainer(env, labelColumn, featureColumn, weights, lossFunction ?? new LogLoss(), learningRate, decreaseLearningRate, l2RegularizerWeight, numIterations);
+        }
+
+        /// <summary>
+        /// Predict a target using a linear binary classification model trained with the AveragedPerceptron trainer, and a custom loss.
```

Comment:
>AveragedPerceptron [](start = 89, length = 18)

<see cref /> on type references.

Reply:
I don't think we need to add a type reference here. Do we ?

My sense is we are using it to refer to the Averaged Perceptron algo, and not to a specific type within ML.NET

---
In reply to: [248543067](https://github.com/dotnet/machinelearning/pull/2163#discussion_r248543067) [](ancestors = 248543067)

Reply:
it says AveragePerception trainer. I think it would be good  to point to the actual trainer class, so they can read more about the algo. Leaving it like this will cause the users to.. web search. 

---
In reply to: [248786650](https://github.com/dotnet/machinelearning/pull/2163#discussion_r248786650) [](ancestors = 248786650,248543067)